### PR TITLE
raven-reader: init at 1.0.72

### PR DESCRIPTION
**Pull request by WolfangAukang**
###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.05 Release Notes (or backporting 21.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2205-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/networking/newsreaders/raven-reader/default.nix [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  name = "${pname}-${version}";
+  src = fetchurl {
+    url = "https://github.com/hello-efficiency-inc/${pname}/releases/download/v${version}/Raven-Reader-${version}.AppImage";
+    sha256 = "sha256-Sx02VMAgmncT9f5Hvs0LugzhD6Z8cWXHx4kn8IG9seU=";
+  };
+  appimageContents = appimageTools.extractType2 { inherit name src; };
+
+in
+appimageTools.wrapType2 {
+  inherit name src;
```
Comment (SuperSandro2000) [inline pkgs/applications/networking/newsreaders/raven-reader/default.nix:15]:
```suggestion
  inherit pname version src;
```

File: pkgs/applications/networking/newsreaders/raven-reader/default.nix [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+  extraInstallCommands = ''
+    mv $out/bin/${name} $out/bin/${pname}
```
Comment (SuperSandro2000) [inline pkgs/applications/networking/newsreaders/raven-reader/default.nix:18]:
```suggestion
    mv $out/bin/${pname}-${version} $out/bin/${pname}
```

File: pkgs/applications/networking/newsreaders/raven-reader/default.nix [hunk 3]
```diff
@@ -0,0 +1,39 @@
+{ lib, fetchurl, appimageTools }:
+
+let
+  pname = "raven-reader";
+  version = "1.0.72";
+  name = "${pname}-${version}";
+  src = fetchurl {
+    url = "https://github.com/hello-efficiency-inc/${pname}/releases/download/v${version}/Raven-Reader-${version}.AppImage";
```
Comment (SuperSandro2000) [inline pkgs/applications/networking/newsreaders/raven-reader/default.nix:8]:
```suggestion
    url = "https://github.com/hello-efficiency-inc/raven-reader/releases/download/v${version}/Raven-Reader-${version}.AppImage";
```

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]

File: pkgs/applications/networking/newsreaders/raven-reader/default.nix (hunk 3 above)
Comment (MoritzBoehme) [reply pkgs/applications/networking/newsreaders/raven-reader/default.nix]: Could you elaborate on why this is better? Isn't it better (less redundancy) or at least as good to substitute with the already defined `pname`?

**Pull request merged**